federation/model/BUILD.bazel

```diff
@@ -5,3 +5,19 @@ go_library(
     srcs = ["bindata.go"],
     visibility = ["//visibility:public"],
 )
+
+genrule(
+  name = "bindata",
+  srcs = glob(["*"], exclude=["bindata.go"]),
```

Comment:
maybe exclude the `BUILD.bazel` file here too

Reply:
Missed that